packages/vite/src/node/constants.ts

```diff
@@ -36,14 +36,9 @@ export const DEFAULT_EXTENSIONS = [
   '.json',
 ]
 
-export const DEFAULT_CONFIG_FILES = [
-  'vite.config.js',
-  'vite.config.mjs',
-  'vite.config.ts',
-  'vite.config.cjs',
-  'vite.config.mts',
-  'vite.config.cts',
-]
+export const DEFAULT_CONFIG_FILES = 'js,mjs,ts,cjs,mts,cts'
+  .split(',')
+  .map((ext) => `vite.config.${ext}`)
```

Comment:
I think it could be
```suggestion
export const DEFAULT_CONFIG_FILES = ['js', 'mjs', 'ts', 'cjs', 'mts', 'cts']
  .map((ext) => `vite.config.${ext}`)
```

But I vote to avoid this change. I prefer the explicit list as it is easier for it to appear in code searches.

Reply:
Yeah I think the previous one was also fine.